data/json/npcs/talk_tags.json

```diff
@@ -1042,7 +1241,10 @@
       "Call the fire department!  Wait, they're dead!  Run away!",
       "This place is on fire.  I'm leaving.",
       "Put the fire out!  Put it out!",
-      "Fire bad! <move>!"
+      "Fire bad! <move>!",
+      "Fire, fire, FIRE!",
+      "Get an extinguisher!",
+      "Danger hot!"
```

Comment:
```suggestion
      "Danger, hot!"
```

Reply:
Danger hot is a military phrase for "Dangerously hot". Danger here is used as an adjective.